mmv1/templates/terraform/update_encoder/managed_dns_zone.go.erb

```diff
@@ -0,0 +1,8 @@
+nameServers, ok := d.GetOkExists("name_servers")
```

Comment:
These are all output-only fields, are we unable to leave them out?

Reply:
Sadly no from my tests. The update API for the managed DNS zone requires the full [object](https://cloud.google.com/dns/docs/reference/v1/managedZones#resource). Otherwise, it'll yield:

```json   
{
  "message": "The 'entity.managedZone.nameServers[1]' parameter is required but was missing.",
  "domain": "global",
  "reason": "required"
},
{
  "message": "The 'entity.managedZone.nameServers[2]' parameter is required but was missing.",
  "domain": "global",
  "reason": "required"
},
{
  "message": "The 'entity.managedZone.nameServers[3]' parameter is required but was missing.",
  "domain": "global",
  "reason": "required"
},
```